.github/workflows/vb_script.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import pennylane as qml
+pl_version = '"' + qml.version() + '"'
+
+
+def bump_version(version_line, pre_release, pl_release):
+    """ A helper function which takes the current version string and
+    replaces it with the bumped version depending on the pre/post
+    release flag.
+
+    Args:
+         version_line (string): The string containing the current
+            version of the plugin.
+         pre_release (bool): A flag which determines if this is a
+            pre-release or post-release version bump.
+        pl_release (bool): A flag which determines if the new
+            version of PennyLane has already been released to Pypi
```

Comment:
Likely, we could do without this option. For the pre-release version bump, there will likely not be a scenario where we have

a) not done the pre-release version bump;
b) have released PennyLane.

Where there's a minor difference in the case of Lightning, is the post-release version bump: we'll likely be releasing PennyLane-Lightning first and then PennyLane. If the opposite order was assumed and used somewhere, then we'd need a change in logic there.

Reply:
Ahh I see. Currently it can support performing both pre or post release version bumps both before or after we have released Pennylane. I can remove this flag as for post-release version bumps I do not use the current version of PL, but instead just increment the version found in the change log and `_version.py` file. My personal recommendation would be to keep it in as this kind of functionality might become useful if in the future we decide to change up when we are going to be releasing some of the plugins.

Reply:
If it's not used at the moment, then maybe it's best to remove it. It can lead to confusion to have an option that is not needed.